samples/matter/lock/src/chip_project_config.h

```diff
@@ -25,3 +25,7 @@
 /* Do not automatically register Thread Network Commissioning instance. */
 #define _NO_NETWORK_COMMISSIONING_DRIVER_
 #endif
+
+#if defined(CONFIG_NCS_SAMPLE_MATTER_SECURE_STORAGE_BACKEND) && defined(CONFIG_LOCK_SCHEDULES)
+#define CHIP_DEVICE_CONFIG_CHIP_TASK_STACK_SIZE 9216
```

Comment:
@kkasperczyk-no recently added the `CHIP_TASK_STACK_SIZE` kconfig so maybe it is worth using it instead of the obsolete definition. If we use the define here the config will be blocked because the `CHIP_DEVICE_CONFIG_CHIP_TASK_STACK_SIZE` is already defined and kconfig has power only if it isn't.

Reply:
Ahh sure, good point.